imagekit-sdk/src/main/java/io/imagekit/sdk/tasks/QueryMaker.java

```diff
@@ -1,20 +1,20 @@
 package io.imagekit.sdk.tasks;
 
 public class QueryMaker {
-    private String query;
+    private final StringBuilder query = new StringBuilder();
 
-    public void put(String q){
-        if (null!=query){
-            query+="&";
+    public void put(String q) {
+        if (query.length() != 0) {
+            query.append("&");
         }
-        else {
-            query="";
-        }
-        query+=q;
+        query.append(q);
     }
 
-    public String  get(){
-        return query;
+    public String get() {
+        return query.toString();
     }
 
+    public String getAsQueryString() {
+        return "?" + get();
+    }
```

Comment:
I don't really like this method

Reply:
What do you suggest? Without this I had to concatenate the "?" where the query string is used which I thought was worse. This is what I mean...what standards are we applying?